pressbooks/pressbooks
# fix: heroicons cache issues

This PR would solve randomly heroicons being not displayed properly after adding more icons to the custom heroicons font.

How to test

1. Checkout this branch
2. Navigate PB interface and review the heroicons are displayed properly
3. Double check the enqueued css has the heroicons url with a v= parameter using an unique ID
